content/en/docs/zero-code/java/quarkus.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Quarkus instrumentation
+linkTitle: Quarkus
+---
+
+## Quarkus OpenTelemetry instrumentation
```

Comment:
The page already has a title, so let's drop this pseudo-title:

```suggestion
```

Otherwise, it looks like this:

> <img width="444" alt="image" src="https://github.com/user-attachments/assets/32f6ee75-e7fa-4313-934b-619fa79eddb5" />